controllers/api/auth_test.go

```diff
@@ -0,0 +1,283 @@
+package api_controllers_test
+
+import (
+	. "go-google-scraper-challenge/helpers/test"
+	"go-google-scraper-challenge/initializers"
+	"net/http"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+type LoginResponse struct {
+	AccessToken  string `json:"access_token"`
+	ExpiresIn    int    `json:"expires_in"`
+	RefreshToken string `json:"refresh_token"`
+	TokenType    string `json:"token_type"`
+}
+
+type ErrorResponse struct {
+	Error            string `json:"error"`
+	ErrorDescription string `json:"error_description"`
+}
+
+var _ = Describe("AuthController", func() {
+	Describe("POST /login", func() {
+		Context("given valid params", func() {
+			It("responses with status ok", func() {
```

Comment:
`It("responses` is not really correct English 😆 

```suggestion
			It("returns with status ok", func() {
```

or:

```suggestion
			It("responds with status ok", func() {
```